homeassistant/components/services.yaml

```diff
@@ -581,30 +610,87 @@ wink:
   refresh_state_from_wink:
     description: Pull the latest states for every device.
 
-homeassistant:
-  check_config:
-    description: Check the Home Assistant configuration files for errors. Errors will be displayed in the Home Assistant log.
-  reload_core_config:
-    description: Reload the core configuration.
-  restart:
-    description: Restart the Home Assistant service. It is normal to get a "Failed to call service homeassistant/restart" message.
-  stop:
-    description: Stop the Home Assistant service. It is normal to get a "Failed to call service homeassistant/stop" message.
-  toggle:
-    description: Generic service to toggle devices on/off under any domain. Same usage as the light.turn_on, switch.turn_on, etc. services.
-    fields:
-      entity_id:
-        description: The entity_id of the device to toggle on/off
-        example: light.living_room
-  turn_on:
-    description: Generic service to turn devices on under any domain. Same usage as the light.turn_on, switch.turn_on, etc. services.
-    fields:
-      entity_id:
-        description: The entity_id of the device to turn on
-        example: light.living_room
-  turn_off:
-    description: Generic service to turn devices off under any domain. Same usage as the light.turn_on, switch.turn_on, etc. services.
-    fields:
-      entity_id:
-        description: The entity_id of the device to turn off
-        example: light.living_room
\ No newline at end of file
+wink_set_siren_volume:
+  description: Set the volume of the siren for a Dome siren/chime.
+
+  fields:
+    entity_id:
+      description: Name(s) of the entities to set
+      example: 'switch.dome_siren'
+    volume:
+        description: Volume level. One of ["low", "medium", "high"]
+        example: "high"
+
+wink_enable_chime:
+  description: Enable the chime of a Dome siren with the provided sound.
+
+  fields:
+    entity_id:
+      description: Name(s) of the entities to set
+      example: 'switch.dome_siren'
+    tone:
+        description: The tone to use for the chime. One of ["doorbell", "fur_elise", "doorbell_extended", "alert", "william_tell", "rondo_alla_turca", "police_siren", "evacuation", "beep_beep", "beep", "inactive"]
+        example: "doorbell"
+
+wink_set_siren_tone:
```

Comment:
In this file all services are actually bound under the key of the domain. However, as this file has been growing, I've been trying to move them away from this file. 

Can you do this:
 - Remove the `wink_` prefix from your services, they are under your component now and so it's fine
 - Move `wink.py` to `wink/__init__.py`
 - Move services to `wink/services.yaml`